<center>
    <img src="https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBMDeveloperSkillsNetwork-DS0105EN-SkillsNetwork/labs/Module2/images/SN_web_lightmode.png" width="300" alt="cognitiveclass.ai logo">
</center>


#### Add your code below following the instructions given in the course


## My Jupyter Notebook on IBM Watson Studio

##### Ilya 
 practical electronics engineer in the present
 Data Science in future 

I really love numbers!
a year ago, when I was studying for a BA in management, I discovered that I liked statistics. In my last year of study, I took 2 courses in data analysis and it was love from the first lesson!
after the first course in analysis, I changed the topic of the scientific work so that data science could be used.
I am now teaching additional classes in R and Python and looking for projects on the topic at my main job.

### 1.Import seaborn,numpy
### 2. Create a variable and read data from seaborn
### 3. Get information about the dataset


In [19]:
import seaborn as sns
import numpy as np
tips=sns.load_dataset("tips")
display(tips.head().T)
display(tips.describe().T)

0       1       2       3       4
total_bill   16.99   10.34   21.01   23.68   24.59
tip           1.01    1.66     3.5    3.31    3.61
sex         Female    Male    Male    Male  Female
smoker          No      No      No      No      No
day            Sun     Sun     Sun     Sun     Sun
time        Dinner  Dinner  Dinner  Dinner  Dinner
size             2       3       3       2       4

count       mean       std   min      25%     50%      75%    max
total_bill  244.0  19.785943  8.902412  3.07  13.3475  17.795  24.1275  50.81
tip         244.0   2.998279  1.383638  1.00   2.0000   2.900   3.5625  10.00
size        244.0   2.569672  0.951100  1.00   2.0000   2.000   3.0000   6.00

# let's analyze the code line by line
##### Import libraries
## 1. [Seaborn](https://seaborn.pydata.org/)
<center>
    <img src="https://seaborn.pydata.org/_static/logo-wide-lightbg.svg" width="300" alt="cognitiveclass.ai logo">
</center>

## 2.[numpy](https://numpy.org/)
<center>
    <img src="https://miro.medium.com/max/765/1*cyXCE-JcBelTyrK-58w6_Q.png" width="300" alt="cognitiveclass.ai logo">
</center>

## 3. loading data into a variable tips 
 [link to Kaggle](https://www.kaggle.com/datasets/ranjeetjain3/seaborn-tips-dataset) 
 
## 4.look at the first 5 lines of df
|      id    |0|1|2|3|4|
|:----------:|:----------:|:----------:|:----------:|:----------:|:--------:|
|total_bill|16.99|10.34|21.01|23.68|24.59|
|tip|1.01|1.66|3.5|3.31|3.61|
|sex|Female|Male|Male|Male|Female|
|smoker	No|No|No|No|No|
|day|Sun|Sun|Sun|Sun|Sun|
|time|Dinner|Dinner|Dinner|Dinner|Dinner|
|size|2|3|3|2|4|

## 5.Generate descriptive statistics

|   |count|mean|std|min|25%|50%|75%|max|
|:----------:|:----------:|:----------:|:----------:|:----------:|:--------:|:--------:|:--------:|:--------:|
|total_bill|244.0|9.785943|8.902412|3.07|13.3475|17.795|24.1275|50.81|
|tip|244.0|2.998279|1.383638	1.00|2.0000|2.900|3.5625|10.00|
|size|244.0|2.569672|0.951100|1.00|2.0000|2.000|3.0000|6.00|